python/dgl/distributed/partition.py

```diff
@@ -187,15 +287,39 @@ def partition_graph(g, graph_name, num_parts, out_path, num_hops=1, part_method=
     If node IDs and edge IDs are not shuffled to ensure that all nodes/edges in a partition
     fall into a contiguous ID range, DGL needs to store node/edge mappings (from
     node/edge IDs to partition IDs) in separate files (node_map.npy and edge_map.npy).
-    The node/edge mappings are stored in numpy files.
+    The node/edge mappings are stored in numpy files. **Note**: this format is
+    deprecated and will not be supported by the next release. In other words, the future
+    release will always shuffle node Ids and edge Ids when partitioning a graph.
+
+    If node Ids and edge Ids are shuffled, `node_map` and `edge_map` contains the information
+    for mapping between global node/edge Ids to partition-local node/edge Ids.
+    For heterogeneous graphs, the information in `node_map` and `edge_map` can also be used
+    to compute node types and edge types. The format of the data in `node_map` and `edge_map`
+    is as follows:
+
+    .. code-block:: none
+
+        {
+            "node_type": [ [ part1_start, part1_end ],
+                           [ part2_start, part2_end ],
+                           ... ],
+            ...
+        },
```

Comment:
remove comma

Reply:
this should be right. there are other items I skipped in the dict.